manager/allocator/cnmallocator/networkallocator.go

```diff
@@ -603,9 +604,16 @@ func (na *cnmNetworkAllocator) allocateVIP(vip *api.Endpoint_VirtualIP) error {
 			return err
 		}
 	}
+	if localNet.nw.IPAM != nil && localNet.nw.IPAM.Driver != nil {
+		opts = localNet.nw.IPAM.Driver.Options
+		if opts == nil {
+			opts = make(map[string]string)
+		}
+		opts[ipamapi.AllocSerialPrefix] = "true"
```

Comment:
Can you check for the presence of this key and not override it if the user explicitly set it to `false` ?
Also, can you pls set this flag only for `default` ipam driver ?

Reply:
This does apply to default ipam driver because of namespaced labels. I can add a check.